lib/src/main/cpp/examples/GetDepthExample.cpp

```diff
@@ -0,0 +1,29 @@
+#include <string>
+#include <fstream>
+#include <vector>
+#include <iostream>
+#include "../include/DepthEstimator.h"
+
+const int kCliArgsNum = 4;
+
+inline std::vector<char> streamToVector(std::ifstream &s) {
+    return {std::istreambuf_iterator<char>(s), std::istreambuf_iterator<char>()};
+}
+
+int main(int argc, char *argv[]) {
+    if (argc != kCliArgsNum) {
+        std::cerr << "Illegal CLI arguments num. Expected " << kCliArgsNum << "(including the command name) but was "
+                  << argc;
+        return 1;
+    }
+
+    auto calibPath = argv[1], leftImgPath = argv[2], rightImgPath = argv[3];
```

Comment:
I'm actually having troubles determining where `const` needs to be placed apart from function parameters. Previously (not in this project) I tended to put const everywhere I could, but then I thought that it just makes code harder to read.

In this particular case I thought it doesn't make much sense since it is just a local variable.